src/main/java/cs2103/v15_1j/jimjim/model/TaskEvent.java

```diff
@@ -21,5 +21,10 @@ public void setName(String name){
 	public StringProperty taskNameProperty() {
 		return name;
 	}
-
+	
+	@Override
+	public boolean equals(Object t) {
```

Comment:
Also, what if 1 Event and 1 FloatingTask have the same name? Are they equal then?